### PR TITLE
feat: add installation steps for popular frameworks

### PR DESCRIPTION
**Pull request by brendan-defi**
**What changed? Why?**
* Added installation steps for OCK's most requested Next.js alternatives:
  * Vite
  * Remix
  * Astro
* Updated `/getting-started` page with the 4 manual installation options (nextjs, vite, remix, astro)
* Created `<Icon>` component in `site` to gracefully handle SVGs

**Notes to reviewers**

**How has it been tested?**
locally

### CODE REVIEW

File: site/docs/components/StartBuilding.tsx [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+          <span>
+            <a
+              className="vocs_Anchor vocs_Link vocs_Link_accent_underlined"
+              href="/token/token-chip"
+            >
+              <code className="vocs_Code">Tokens</code>
+            </a>{' '}
+            - Search and display{' '}
+            <a
+              className="vocs_Anchor vocs_Link vocs_Link_accent_underlined"
```
Comment (alessey) [inline site/docs/components/StartBuilding.tsx:128]: i don't necessarily dislike it, but the hover on these links isn't used anywhere else

Comment (brendan-defi) [reply site/docs/components/StartBuilding.tsx:128]: sorry, i didn't notice that these are currently wrapped in `<strong>` tags. that removes the hover. will update.

**Review by 0xAlec — APPROVED** [review]

File: site/docs/pages/installation/nextjs.mdx [hunk 2]
```diff
@@ -0,0 +1,228 @@
+---
+title: Astro Installation · OnchainKit
+description: Install OnchainKit using Astro
```
Comment (dschlabach) [inline site/docs/pages/installation/nextjs.mdx:3]: Next

File: site/docs/components/Icon.tsx [hunk 3]
```diff
@@ -0,0 +1,169 @@
+export type IconProps = {
```
Comment (cpcramer) [inline site/docs/components/Icon.tsx:1]: I think we should handle these SVGs similar to how we organize and access them in `/src` 

Icon folder with each SVG in its own file - `astroSVG.ts` etc

File: site/sidebar.ts [hunk 4]
```diff
@@ -3,7 +3,19 @@ import type { Sidebar } from 'vocs';
 export const sidebar = [
   {
     text: 'Introduction',
-    items: [{ text: 'Getting Started', link: '/getting-started' }],
+    items: [
+      { text: 'Getting Started', link: '/getting-started' },
+      // { text: 'Installation', link: '/installation' },
```
Comment (dschlabach) [inline site/sidebar.ts:8]: remove?

File: site/docs/components/Icon.tsx (hunk 3 above)
Comment (cpcramer) [reply site/docs/components/Icon.tsx:1]: There is a` site/docs/components/svgs `folder already https://github.com/coinbase/onchainkit/blob/paul/update-maxSlipapge/site/docs/components/svg/fundSvg.tsx#L1

File: site/docs/pages/installation/nextjs.mdx (hunk 2 above)
Comment (brendan-defi) [reply site/docs/pages/installation/nextjs.mdx:3]: oops, thank you!

File: site/sidebar.ts (hunk 4 above)
Comment (brendan-defi) [reply site/sidebar.ts:8]: yeah, oversight, thank you

File: site/docs/components/Icon.tsx (hunk 3 above)
Comment (cpcramer) [reply site/docs/components/Icon.tsx:1]: Also try to use `ock-fill-*` variables for styling so the icons will update during theme changes!

File: site/docs/components/InstallationOptions.tsx [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+  const { theme } = useTheme();
+  return (
+    <Link
+      to={href}
+      className="m-2 rounded-md border-2 border-zinc-300 text-zinc-950 hover:bg-zinc-100 dark:border-zinc-700 dark:text-zinc-50 dark:hover:bg-zinc-900"
+    >
+      <div className="flex flex-col items-center gap-2 py-10">
+        <Icon
+          name={name.replace('.', '').toLowerCase()}
+          color={theme === 'dark' ? 'white' : 'black'}
```
Comment (cpcramer) [inline site/docs/components/InstallationOptions.tsx:31]: Updating the SVG classes directly will automatically update this

File: site/docs/components/Icon.tsx (hunk 3 above)
Comment (brendan-defi) [reply site/docs/components/Icon.tsx:1]: okay, i'll give this a shot.

File: site/docs/components/InstallationOptions.tsx (hunk 5 above)
Comment (brendan-defi) [reply site/docs/components/InstallationOptions.tsx]: cool, got this working, thanks for the guidance!

**Review by dschlabach — APPROVED** [review]

**Pull request merged**